.github/workflows/release.yml

```diff
@@ -1,36 +1,20 @@
+name: Release sonar-dummy-js
+
 on:
   release:
-    types:
-      - published
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Version'
+        required: false
 
 jobs:
-  publish:
-    runs-on: ubuntu-latest
-    env:
-      RELEASE_TAG: ${{ github.event.release.tag_name }}
-      NPM_REPOSITORY: "sonarsource-npm-public"
-      SCOPE: "%40sonarsource/"
-      PACKAGE: "sonar-dummy-js"
+  releaseasing:
+    uses: SonarSource/gh-action_release/.github/workflows/main.yaml@5.9.4 #TODO: update to latest version
```

Comment:
TODO yes :) 

small reminder: do not forget to use <SHA> # 5.9.4 format not @version to ensure immutability of the release